android/src/main/java/com/brentvatne/receiver/AudioBecomingNoisyReceiver.java

```diff
@@ -26,7 +25,7 @@ public void onReceive(Context context, Intent intent) {
     public void setListener(BecomingNoisyListener listener) {
         this.listener = listener;
         IntentFilter intentFilter = new IntentFilter(AudioManager.ACTION_AUDIO_BECOMING_NOISY);
-        ContextCompat.registerReceiver(context, this, intentFilter, ContextCompat.RECEIVER_NOT_EXPORTED);
```

Comment:
Can you revert this change, it is recent to fix a crash when targeting api 34 !

Reply:
I revert changed. also revert code related androidx.core:core dependency